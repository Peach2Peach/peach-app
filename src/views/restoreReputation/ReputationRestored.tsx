import { View } from 'react-native'
import { Icon, Text } from '../../components'

import tw from '../../styles/tailwind'
import i18n from '../../utils/i18n'

export const ReputationRestored = () => (
  <View style={tw`items-center justify-center h-full`}>
    <Text style={tw`text-center h4 text-primary-background-light`}>{i18n('restoreBackup.reputationRestored')}</Text>
    <Text style={tw`text-center body-l text-primary-background-light`}>{i18n('restoreBackup.welcomeBack')}</Text>
    <Icon id="save" style={tw`w-32 h-32 mt-16`} color={tw.color('primary-background-light')} />
  </View>
)
