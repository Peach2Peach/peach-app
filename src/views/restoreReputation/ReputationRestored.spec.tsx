import { createRenderer } from 'react-test-renderer/shallow'
import { ReputationRestored } from './ReputationRestored'

describe('ReputationRestored', () => {
  const shallowRenderer = createRenderer()

  it('should render correctly', () => {
    shallowRenderer.render(<ReputationRestored />)
    expect(shallowRenderer.getRenderOutput()).toMatchSnapshot()
  })
})
