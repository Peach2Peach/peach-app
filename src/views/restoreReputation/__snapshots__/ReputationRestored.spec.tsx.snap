// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReputationRestored should render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "display": "flex",
      "height": "100%",
      "justifyContent": "center",
    }
  }
>
  <PeachText
    style={
      {
        "color": "#FFFCFA",
        "fontFamily": "Baloo2-Bold",
        "fontSize": 34,
        "letterSpacing": 0.34,
        "lineHeight": 42,
        "textAlign": "center",
      }
    }
  >
    reputation restored
  </PeachText>
  <PeachText
    style={
      {
        "color": "#FFFCFA",
        "fontFamily": "Baloo2-Regular",
        "fontSize": 20,
        "letterSpacing": 0.2,
        "lineHeight": 28,
        "textAlign": "center",
      }
    }
  >
    welcome back!
  </PeachText>
  <Icon
    color="#FFFCFA"
    id="userCheck"
    style={
      {
        "height": 128,
        "marginTop": 64,
        "width": 128,
      }
    }
  />
</View>
`;
